torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -934,6 +913,28 @@ class TensorExprFuser {
           return false;
         }
 
+        // Operator is only supported on CUDA.
+        if (node->isMemberOf(cuda_only_operator_set)) {
```

Comment:
could we move this outside of `for`, otherwise we seem to be re-checking for every value?

Reply:
we omitted checking the output types which is probably okay since these ops aren't c-tor ones.